image/copy-files.sh

```diff
@@ -0,0 +1,10 @@
+sudo mkdir -p /etc/containers
```

Comment:
Could we have a comment saying this file purpose: to be used by packer to copy files over the VM filesystem?